src/ceph-daemon

```diff
@@ -558,6 +557,7 @@ def deploy_daemon_units(fsid, uid, gid, daemon_type, daemon_id, c,
     call(['systemctl', 'reset-failed', unit_name],
          verbose_on_failure=False)
     if enable:
+        call_throws(['systemctl', 'link', unit_file])
```

Comment:
is 'link' a no-op when it's in /etc/systemd/system?

Reply:
hmm, it does fail if the link already exists

It feels a little dirty, but we could maybe do this:
```
        call(['systemctl', 'link', unit_file],$
             verbose_on_failure=False)$
```

Reply:
Per this comment -- https://github.com/ceph/ceph/pull/31486#discussion_r344218007

Refactoring the standalone tests to not intermix the src and dst targets appears to remove the need to (re)link the systemd unit